assets/js/modules/search-console/util/report-date-range-args.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Report date range args utils.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Generates an object with the appropriate structure to pass to report selectors.
```

Comment:
```suggestion
 * Generates a date range object with the appropriate structure to pass to report selectors.
```

Let's be specific about the kind of objects and/or reports we're talking about—this docstring will lack a bit of context without that.